### ISSUE TITLE
Title: WCAG 1.1.1: Ensures <img> elements have alternate text or a role of none or presentation (img)

### ISSUE THREAD

**Issue by A11Ybot-OP** [knative/ux#64]
Tags: Accessibility, WCAG 1.1.1, image-alt

Issue: Ensures <img> elements have alternate text or a role of none or presentation 

Target application: Home - Knative - https://knative.dev/docs/

Element path: .animated-gradient > img

Snippet: <img src="images/logo/rgb/knative-logo-rgb.png" style="display: none;">

How to fix: 
Fix any of the following:
  Element does not have an alt attribute
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Chrome version 97.0.4692.99
